### ISSUE TITLE
Update flatpak version

### ISSUE THREAD

**Issue by Ristovski** [mono/monodevelop#9609]
The flatpak version of MonoDevelop has not been updated in over 3 years now.

Are there any plans on updating it?
Alternatively, an official AppImage would be a nice solution as well.

**Comment by decriptor** [comment]
@directhex

**Comment by directhex** [comment]
Nah. I'll be honest, I got so much abuse from people because the inadequate sandbox handling got in the way for too many people (not a single patch offered though, of course), that I lost any desire to continue on it. I have other things I can spend my time on.

**Comment by PietroMB** [comment]
oh pls, appimage have no sanbox! your work is awsome pls let us install it as appimage and let us be free from apt milion multiple repos
